.travis.yml

```diff
@@ -41,6 +40,48 @@ matrix:
   include:
     - arch: s390x
       jdk: openjdk11
+    - dist: focal
```

Comment:
The default dist configured above we use is `trusty`. Should we upgrade the default dist to `focal`? (it can be a separate work)

Reply:
Currently, the "arm64-graviton2" resources are only available in `Xenial`, `Bonic`, `Focal` dists of Ubuntu, please see the [Travis doc](https://docs.travis-ci.com/user/reference/overview#which-one-do-i-use)， I just picked a lastest version.

Reply:
Yeah, I was thinking maybe we can simply this by updating https://github.com/apache/storm/blob/fe2f7102e244336e288d26f2dde8089198ee4c33/.travis.yml#L28

Reply:
OK, let's me try